pallets/admin-utils/tests/tests.rs

```diff
@@ -1225,39 +1228,37 @@ fn test_sudo_get_set_alpha() {
 
         let hotkey: U256 = U256::from(1);
         let coldkey: U256 = U256::from(1 + 456);
-        let signer = <<Test as Config>::RuntimeOrigin>::signed(coldkey.clone());
+        let signer = <<Test as Config>::RuntimeOrigin>::signed(coldkey);
 
         // Enable Liquid Alpha and setup
         SubtensorModule::set_liquid_alpha_enabled(netuid, true);
         migration::migrate_create_root_network::<Test>();
-        SubtensorModule::add_balance_to_coldkey_account(
-            &coldkey,
-            1_000_000_000_000_000,
-        );
-        assert_ok!(SubtensorModule::root_register(
-            signer.clone(),
-            hotkey,
-        ));
-        assert_ok!(SubtensorModule::add_stake(
-            signer.clone(),
-            hotkey,
-            1000
-        ));
+        SubtensorModule::add_balance_to_coldkey_account(&coldkey, 1_000_000_000_000_000);
+        assert_ok!(SubtensorModule::root_register(signer.clone(), hotkey,));
+        assert_ok!(SubtensorModule::add_stake(signer.clone(), hotkey, 1000));
 
         // Should fail as signer does not own the subnet
         assert_err!(
             AdminUtils::sudo_set_alpha_values(signer.clone(), netuid, alpha_low, alpha_high),
             DispatchError::BadOrigin
         );
 
-        assert_ok!(SubtensorModule::register_network(
-            signer.clone()
-        ));
+        assert_ok!(SubtensorModule::register_network(signer.clone()));
 
-        assert_ok!(AdminUtils::sudo_set_alpha_values(signer.clone(), netuid, alpha_low, alpha_high));
-        let (grabbed_alpha_low, grabbed_alpha_high): (u16, u16) = SubtensorModule::get_alpha_values(netuid);
+        assert_ok!(AdminUtils::sudo_set_alpha_values(
+            signer.clone(),
+            netuid,
+            alpha_low,
+            alpha_high
+        ));
+        let (grabbed_alpha_low, grabbed_alpha_high): (u16, u16) =
```

Comment:
This is the line that I'm referring to @distributedstatemachine

Reply:
In test_clamp_alpha_values(), we call the "get_alpha_value" function to get the "grabbed_alpha_low", there is such analogous call in for "alpha" test_compute_alpha_values_256_miners()

Reply:
As explained in discord , this is a different test and isnt testing the calculation of alpha values , but setting them with the `sudo_set_alpha_values` admin extrinsic

Reply:
https://github.com/opentensor/subtensor/pull/552/files/2e51ebdf446ce58fd2ca81e72329354e2955bcdb..b8364014f36b6a72357881cacb0f3462a3fde5a0#diff-a89ee0066188e9aac5a009116c3c587c2fe6d6b4eff0dfd66bf50c00e0c82350R1223

Reply:
Got it, it would be great to add a note in comments to let the reader know the scope of the test, i.e., "setting values" versus testing the calculation of alpha values.